contributed_definitions/nyaml/NXmpes.yaml

```diff
@@ -135,107 +337,345 @@ NXmpes(NXobject):
         exists: optional
         doc: |
           Manipulator for positioning of the sample.
-        sample_temperature(NX_FLOAT):
+        temperature_sensor(NXsensor):
           exists: recommended
-          unit: NX_TEMPERATURE
-        drain_current(NX_FLOAT):
+          name:
+            exists: recommended
+          measurement:
+            enumeration: [temperature]
+          type:
+            exists: optional
+          value(NX_FLOAT):
+            exists: recommended
+        sample_heater(NXactuator):
           exists: recommended
-          unit: NX_CURRENT
-        sample_bias(NX_FLOAT):
+          name:
+            exists: recommended
+          physical_quantity:
+            enumeration: [temperature]
+          type:
+            exists: optional
+          heater_power(NX_FLOAT):
+            exists: recommended
+          (NXpid):
+            exists: recommended
+            setpoint(NX_FLOAT):
+              exists: recommended
+        drain_current_amperemeter(NXsensor):
+          exists: recommended
+          name:
+            exists: recommended
+          measurement:
+            enumeration: [current]
+          type:
+            exists: optional
+          value(NX_FLOAT):
+            exists: recommended
+        sample_bias_voltmeter(NXsensor):
+          exists: recommended
+          name:
+            exists: recommended
+          measurement:
+            enumeration: [voltage]
+          type:
+            exists: optional
+          value(NX_FLOAT):
+            exists: recommended
+        device_information(NXfabrication):
+          exists: recommended
+          vendor:
+            exists: recommended
+          model:
+            exists: recommended
+          identifier:
+      pressure_gauge(NXsensor):
+        exists: recommended
+        doc: |
+          Device to measure the gas pressure around the sample.
+        name:
+          exists: recommended
+        measurement:
+          enumeration: [pressure]
+        type:
+          exists: optional
+        value(NX_FLOAT):
+          exists: recommended
+          unit: NX_PRESSURE
+          doc: |
+            In case of a single or averaged gas pressure measurement, this is the scalar gas pressure around
+            the sample. It can also be an 1D array of measured pressures (without time stamps).
+        value_log(NXlog):
+          exists: optional
+          value(NX_NUMBER):
+            unit: NX_PRESSURE
+            doc: |
+              In the case of an experiment in which the gas pressure changes and is recorded,
+              this is an array of length m of gas pressures.
```

Comment:
I now remember we made value recommended because there is also value_log. But I still think a sensor should have a value. If it additionally has a log, the value can be the array w/o timestamps, or the first value or the mean, but it should not be empty.

Reply:
Made all ` value` fields required.